go/test/endtoend/backup/vtctlbackup/backup_utils.go

```diff
@@ -413,6 +454,11 @@ func stopAllTablets() {
 	var mysqlProcs []*exec.Cmd
 	for _, tablet := range []*cluster.Vttablet{master, replica1, replica2} {
 		tablet.VttabletProcess.TearDown()
+		if tablet.MysqlctldProcess.TabletUID > 0 {
+			tablet.MysqlctldProcess.Stop()
+			localCluster.VtctlclientProcess.ExecuteCommand("DeleteTablet", "-allow_master", tablet.Alias)
+			continue
+		}
```

Comment:
Same, we should do if/else on mysqlctl and mysqlctld

Reply:
as we are continue stmt at end-if we don't need else stmt